nss-day-cohort-39/tabloid-full-stack-velours
# admin deactivation protections

### Changes

- Merged lastest master (1:08pm)
- Added protections to prevent admins from deactivating themselves
- Added protections to prevent fewer than 2 admins from being deactivated

### Testing
- Login in as admin
- Try to edit or deactivate yourself and make sure a window pops up preventing it
- Try to activate fewer than 2 admins (including yourself) and make sure an error window pops up preventing it
- Make sure same code does not apply to authors
